src/Chainweb/Version.hs

```diff
@@ -564,12 +564,15 @@ window Development = Just $ WindowWidth 120
 window Testnet03 = Just $ WindowWidth 120
 window Mainnet01 = Just $ WindowWidth 120
 
-txSilenceEndDate :: ChainwebVersion -> Maybe (Time Micros)
-txSilenceEndDate Test{} = Nothing
-txSilenceEndDate TimedConsensus{} = Nothing
-txSilenceEndDate PowConsensus{} = Nothing
-txSilenceEndDate TimedCPM{} = Nothing
-txSilenceEndDate FastTimedCPM{} = Nothing
-txSilenceEndDate Development = Nothing
-txSilenceEndDate Testnet03 = Nothing
-txSilenceEndDate Mainnet01 = Just [timeMicrosQQ| 2019-12-05T00:00:00.0 |]
```

Comment:
No longer used.

Reply:
This must be reverted. We can't change history. These rules existed to validate the history of the block chains. They should stay forever.

Reply:
I propose that we keep this (date `d1`) to not change existing validation rules) and define a new constant for date `d2`:

```Haskell
softTxSilenceEndDate Mainnet01 = Just [timeMicrosQQ| 2019-12-07T00:00:00.0 |]
```

Which will be used to implement step (2) from the timeline.